### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rhatdan**
Add commands section
Add information about skopeo sync

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/skopeo/issues/63

**Comment by rhatdan** [comment]
@vrothberg @mtrmac @QiWang19 @TomSweeneyRedHat PTAL

File: README.md [hunk 1]
```diff
@@ -189,6 +195,20 @@ Contributing
...
+| [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry                                |
+| [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags the transport-specific image repository.                               |
+| [skopeo-login(1)](/docs/skopeo-login.1.md)         | Login to a container registry.                                                               |
+| [skopeo-logout(1)](/docs/skopeo-logout.1.md)       | Logout of a container registry.                                                              |
+| [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest of manifest-file and write it to standard output.   |
+| [skopeo-standalone-sign(1)](/docs/skopeo-standalone-sign.1.md)    | Simple sign an image.                                                         |
+| [skopeo-standalone-verify(1)](/docs/skopeo-standalone-verify.1.md)| Verify an image signature.                                                    |
```
Comment (mtrmac) [inline README.md:209]: I’d prefer to not promote the three above much. (OTOH they are not _completely hidden_ like `skopeo layers`.)

(Would it make sense to have this above the examples, or combine it with the “ Skopeo can perform operations which consist of:” list? That can clearly be deferred to a different PR.)

Comment (rhatdan) [reply README.md:209]: If we want to deprecate the use of these, we should hide them from users, and stop displaying them from skopeo --help.

But lets get this merged and then we can discuss removing the visibility of these commands

File: README.md [hunk 2]
```diff
@@ -189,6 +195,20 @@ Contributing
 
 Please read the [contribution guide](CONTRIBUTING.md) if you want to collaborate in the project.
 
+## Commands
+| Command                                            | Description                                                                                  |
+| -------------------------------------------------- | ---------------------------------------------------------------------------------------------|
+| [skopeo-copy(1)](/docs/skopeo-copy.1.md)           | Copy an image (manifest, filesystem layers, signatures) from one location to another.        |
+| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark image-name for deletion.                                                                |
+| [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry                                |
```
Comment (TomSweeneyRedHat) [inline README.md:203]:
```suggestion
| [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry.                                |
```

File: README.md [hunk 3]
```diff
@@ -189,6 +195,20 @@ Contributing
...
+| [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags the transport-specific image repository.                               |
```
Comment (TomSweeneyRedHat) [inline README.md:204]:
```suggestion
| [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags for the transport-specific image repository.                               |
```

File: README.md [hunk 4]
```diff
@@ -189,6 +195,20 @@ Contributing
...
+| [skopeo-login(1)](/docs/skopeo-login.1.md)         | Login to a container registry.                                                               |
+| [skopeo-logout(1)](/docs/skopeo-logout.1.md)       | Logout of a container registry.                                                              |
+| [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest of manifest-file and write it to standard output.   |
+| [skopeo-standalone-sign(1)](/docs/skopeo-standalone-sign.1.md)    | Simple sign an image.                                                         |
```
Comment (TomSweeneyRedHat) [inline README.md:208]:
```suggestion
| [skopeo-standalone-sign(1)](/docs/skopeo-standalone-sign.1.md)    | Debugging tool - Publish and sign an image in one step.                                                         |
```

File: README.md [hunk 5]
```diff
@@ -189,6 +195,20 @@ Contributing
...
+| [skopeo-copy(1)](/docs/skopeo-copy.1.md)           | Copy an image (manifest, filesystem layers, signatures) from one location to another.        |
+| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark image-name for deletion.                                                                |
+| [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry                                |
+| [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags the transport-specific image repository.                               |
+| [skopeo-login(1)](/docs/skopeo-login.1.md)         | Login to a container registry.                                                               |
+| [skopeo-logout(1)](/docs/skopeo-logout.1.md)       | Logout of a container registry.                                                              |
+| [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest of manifest-file and write it to standard output.   |
```
Comment (TomSweeneyRedHat) [inline README.md:207]: I must have missed this when it came in for a review, and I see skopeo-manifest-digest(1) has the same line in it's title.  Does this make more sense?
```suggestion
| [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest for a manifest-file and write it to standard output.   |
```

File: README.md [hunk 6]
```diff
@@ -189,6 +195,20 @@ Contributing
 
 Please read the [contribution guide](CONTRIBUTING.md) if you want to collaborate in the project.
 
+## Commands
+| Command                                            | Description                                                                                  |
+| -------------------------------------------------- | ---------------------------------------------------------------------------------------------|
+| [skopeo-copy(1)](/docs/skopeo-copy.1.md)           | Copy an image (manifest, filesystem layers, signatures) from one location to another.        |
+| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark image-name for deletion.                                                                |
```
Comment (TomSweeneyRedHat) [inline README.md:202]: Side note, the skopeo-delete(1) page shows a `--force` option in the examples section, but it is not listed as an option in the options section.
Maybe:
```suggestion
| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark the image-name for later deletion by the registry's garbage collector.                                                                |
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
I'm done picking, thanks for working this up @rhatdan

**Review by rhatdan — COMMENTED** [review]
Taking suggestions.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL again.

File: README.md (hunk 4 above)
Comment (mtrmac) [reply README.md]: That’s not what the command does; it only creates a local file from local files with no registry interaction at all.

File: README.md (hunk 6 above)
Comment (mtrmac) [reply README.md]: The primary user-relevant outcome is that the image may be _immediately_ impossible to pull. GC is a secondary internal implementation details only server administrators care about.

Comment (TomSweeneyRedHat) [reply README.md]: FWIW, I picked this and the other point based on info on the respective man pages.  Perhaps they need embellishing too?

**Comment by TomSweeneyRedHat** [comment]
LGTM
@mtrmac any objections to merging now and tweaking later?

File: docs/skopeo-inspect.1.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 % skopeo-inspect(1)
 
 ## NAME
-skopeo\-inspect - Return low-level information about _image-name_ in a registry
+skopeo\-inspect - Return low-level information about _image-name_ in a registry.
```
Comment (mtrmac) [inline docs/skopeo-inspect.1.md:4]: For future reference, man-pages(7), man(7) and ultimately lexgrog(1) all suggest that the NAME section should not end with a period.

But man pages are inconsistent in practice, and I certainly don’t care enough to create — or review — a PR for that.
